src/components/dialog/__spec__.js

```diff
@@ -515,58 +432,80 @@ describe('Dialog', () => {
         );
       });
 
-      it('renders a role attribute from the ariaRole prop', () => {
-        expect(wrapper.find('[role="dialog"]').exists()).toBe(false);
+      it('does not render a role attribute from the ariaRole prop, aria-labelledby pointing at the title element or an aria-describedby attribute pointing at the subtitle element', () => {
+        expect(wrapper).toMatchSnapshot();
       });
+    });
 
-      it('renders an aria-labelledby pointing at the title element', () => {
-        expect(wrapper.find('[aria-labelledby="carbon-dialog-title"]').exists()).toBe(false);
-      });
+    describe('focus', () => {
+      let wrapper;
+
+      beforeEach(() => {
+        const mockWindow = {
+          addEventListener() { },
+          removeEventListener() { },
+          getComputedStyle() { return {}; }
+        };
 
-      it('renders an aria-describedby attribute pointing at the subtitle element', () => {
-        expect(wrapper.find('[aria-describedby="carbon-dialog-subtitle"]').exists()).toBe(false);
+        Browser.getWindow = jest.fn();
+        Browser.getWindow.mockReturnValue(mockWindow);
+
+        wrapper = mount(
+          <Dialog
+            onCancel={() => { }}
+            onConfirm={() => { }}
+            showCloseIcon
+            open
+            subtitle='Test'
+            title='Test'
+            ariaRole='dialog'
+          />
+        );
       });
-    });
 
-    describe('when autoFocus is true', () => {
-      it('focuses on the dialog when opened', () => {
-        wrapper.setProps({
-          open: false,
-          autoFocus: true
-        });
-        instance = wrapper.instance();
-        instance.focusDialog = jest.fn();
+      describe('when autoFocus is true', () => {
+        it('focuses on the dialog when opened', () => {
+          wrapper.setProps({
+            open: false,
+            autoFocus: true
+          });
+          instance = wrapper.instance();
+          instance.focusDialog = jest.fn();
```

Comment:
[Minor] Formatting seems slightly off

Reply:
I *think* it's correct Ian. It wasn't picked up by the linter. It's possibly because a couple of those specs have been nested deeper. https://github.com/Sage/carbon/blob/1260dfbc6da113b81112216c73cc580360b09c54/src/components/dialog/__spec__.js#L466-L496

Reply:
Yep, my mistake.